app/services/TtpService.scala

```diff
@@ -234,4 +242,25 @@ object TtpService {
     case someAmount: UpfrontPaymentAnswers.DeclaredUpfrontPayment => Some(someAmount.amount)
     case UpfrontPaymentAnswers.NoUpfrontPayment                   => None
   }
+
+  /**
+   * If email matches the one from the eligibility API - ETMP
+   * If new email entered on the screen which doesn't match the ETMP one - TEMP
+   */
+  private def deriveCustomerDetail(journey: Journey.Stages.EmailVerificationComplete): Option[List[CustomerDetail]] = {
+    val etmpEmails: Option[List[CustomerDetail]] =
+      journey.eligibilityCheckResult.customerDetails.map(_.filter(_.emailSource.contains(EmailSource.ETMP)))
+
+    val emailThatsBeenVerified: String = journey.emailVerificationAnswers match {
```

Comment:
I think you can just do `journey.emailToBeVerified` here to get the email address

Reply:
ok